data/data/azure/master/master.tf

```diff
@@ -0,0 +1,86 @@
+resource "azurerm_network_interface" "master" {
+  count               = "${var.instance_count}"
+  name                = "${var.cluster_id}-master-nic-${count.index}"
+  location            = "${var.region}"
+  resource_group_name = "${var.resource_group_name}"
+
+  ip_configuration {
+    subnet_id                     = "${var.subnet_id}"
+    name                          = "master-${count.index}"
+    private_ip_address_allocation = "Static"
+    private_ip_address            = "${cidrhost(var.master_subnet_cidr, 5 + count.index)}" # azure reserves first 3 ip, 4th is for bootstrap VM, so we start at 5
+  }
+}
+
+resource "azurerm_network_interface_backend_address_pool_association" "master" {
+  count                   = "${var.instance_count}"
+  network_interface_id    = "${element(azurerm_network_interface.master.*.id, count.index)}"
+  backend_address_pool_id = "${var.elb_backend_pool_id}"
+  ip_configuration_name   = "master-${count.index}"                                          #must be the same as nic's ip configuration name.
```

Comment:
this should be pipConfig to match cluster api

Reply:
fixed